extensions/empty-screenshots/src/index.ts

```diff
@@ -2,22 +2,50 @@ import { readdir, stat, rm } from "fs/promises";
 import { join, resolve } from "path";
 import { homedir } from "os";
 import { showToast, Toast, showHUD, getPreferenceValues } from "@raycast/api";
+import { exec } from "child_process";
 
 interface Preferences {
+  screenshotBehavior: string;
   daysToKeep: string;
   screenshotFolder: string;
+  prefixString: string;
+}
+
+async function moveToTrash(filePath: string) {
+  return new Promise((resolve, reject) => {
+    exec(`osascript -e 'tell application "Finder" to delete POSIX file "${filePath}"'`, (error, stdout, stderr) => {
+      if (error) {
+        console.error("Error moving file to Trash:", stderr);
+        reject(error);
+      } else {
+        console.log("File moved to Trash:", stdout);
+        resolve(stdout);
+      }
+    });
+  });
 }
 
 export default async function main() {
-  const { daysToKeep, screenshotFolder } = getPreferenceValues<Preferences>();
+  const { daysToKeep, screenshotFolder, screenshotBehavior, prefixString } = getPreferenceValues<Preferences>();
   const numberOfDays = parseInt(daysToKeep, 10);
-  const folder = resolve(screenshotFolder.replace("~", homedir));
+  const folder = resolve(screenshotFolder.replace("~", homedir()));
 
   try {
     const files = await readdir(folder);
 
     const screenshots = files
-      .filter((file) => file.endsWith(".png") || file.endsWith(".mov"))
+      .filter((file) => {
+        const endsWithValidExtension = file.endsWith(".png") || file.endsWith(".mov");
+        let startsWithValidPrefix = false;
+
+        if (prefixString) {
+          startsWithValidPrefix = file.startsWith(prefixString);
+        } else {
+          startsWithValidPrefix = file.startsWith("CleanShot") || file.startsWith("Screen");
+        }
+
+        return endsWithValidExtension && startsWithValidPrefix;
+      })
```

Comment:
Could we split these into two separate `.filter` functions?
```suggestion
      .filter((file) => file.endsWith(".png") || file.endsWith(".mov"))
      .filter((file) => prefixString ? file.startsWith(prefixString) : file.startsWith("CleanShot") || file.startsWith("Screen"))
```

Reply:
Additionally, with the default behavior of this filter (`file.startsWith("CleanShot") || file.startsWith("Screen")`), should we assume all screenshots start with `CleanShot` or `Screen`? This might break existing installations of the extension if we assume that to be true.

I think since we are targeting an additive behavior here, we should probably default to the existing behavior of not caring about the prefix, and with configuration value take a comma separated list of prefix values. This way the config could be `CleanShot, Screen` and we could do `prefixString.replace(' ', '').split(',')` and filter by that instead.